### PR TITLE
Streaming Page

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (485)

In this PR, I have added single video streaming feature and also changed the UI of video card a bit.

### CODE REVIEW

File: src/pages/homepage.jsx [hunk 1]
```diff
@@ -8,7 +9,7 @@ import { useAuth } from "../context/authContext";
 
 const Homepage = () =>{
     const [sidebar, setSidebar] = useState(true);
-    const [ drop, setDrop ] = useState(false);
+    // const [ drop, setDrop ] = useState(false);
```
Comment (zoyatanweer) [inline src/pages/homepage.jsx:13]: remove comments

File: src/pages/homepage.jsx [hunk 2]
```diff
@@ -37,49 +42,40 @@ return (
...
-                <article className="video-card">
-                    <img src={video.cover} alt="" className="card-img" onClick={()=>getHistory(token, video)}/>
+                <article className="video-card" key={video._id}>
+                    <Link to={`/homepage/${video._id}`}>
+                    <img src={video.cover} alt="" className="card-img" onClick={()=>getHistory(token, video)}/></Link>
                     <div className="content">
                         <div className="card-head">
                             <p className="title" onClick={()=>getHistory(token, video)}>{video.title}</p>
-                            <span className="dp-span">{ drop === true ? (
+                            {/* <span className="dp-span">{ drop === true ? (
```
Comment (zoyatanweer) [inline src/pages/homepage.jsx:62]: remove commented code

File: src/styles/homepage.css [hunk 3]
```diff
@@ -140,4 +141,16 @@
...
     text-decoration: underline;
     cursor: pointer;
+}
+.card-icons{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 5px;
+    background-color: #24292c;
```
Comment (zoyatanweer) [inline src/styles/homepage.css:152]: use var for color

File: src/pages/homepage.jsx [hunk 4]
```diff
@@ -26,6 +27,10 @@ videoState.liked.some((item) => item._id === video._id) ?
 removeLikes(token, video._id) : getLikes(token, video)
 }
 
+const playVideoHandler = (token, video) =>{
+    getHistory(token, video)
+    
+}
```
Comment (omjadhav85) [inline src/pages/homepage.jsx:35]: I think this is a reduntant function as I dont see it getting called anywhere, Can be removed?

File: src/styles/stream.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
+.player-div{
+    padding: 10px;
+    display: flex;
+    margin: 0px 20px 20px 20px;
+    flex-direction: column;
+    width: 100%;
+    background-color: #24292c;
```
Comment (zoyatanweer) [inline src/styles/stream.css:7]: use var for color

File: src/styles/stream.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+}
+
+.video-details{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* justify-content: space-between; */
```
Comment (zoyatanweer) [inline src/styles/stream.css:17]: remove commentd

File: src/styles/stream.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    gap: 20px;
+    text-align: left;
+    width: 78.5%;
+    color: #a0b2b9;
```
Comment (zoyatanweer) [inline src/styles/stream.css:21]: use var

File: src/styles/stream.css [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    padding: 10px;
+    background-color: #12191d;
```
Comment (zoyatanweer) [inline src/styles/stream.css:23]: use var

File: src/styles/stream.css [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+.title-div{
+    justify-items: center;
+    border-bottom: 1px solid #a0b2b9;
```
Comment (zoyatanweer) [inline src/styles/stream.css:29]: use var

File: src/styles/stream.css [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+    /* width: 100%; */
```
Comment (zoyatanweer) [inline src/styles/stream.css:30]: use var

**Review by omjadhav85 — APPROVED** [review]
Code looks good as easy to understand! Some minor refactoring suggested. Else its good to merge!

**Review by zoyatanweer — APPROVED** [review]
- clean and good code
- do the suggested changes 
- Awesome UI, looks very beautiful
- good to merge

File: src/pages/homepage.jsx (hunk 4 above)
Comment (Jia-A) [reply src/pages/homepage.jsx:35]: Oh yes, forgot to remove it.

File: src/styles/stream.css (hunk 10 above)
Comment (Jia-A) [reply src/styles/stream.css:30]: Will work on the suggestions

**Pull request merged**